testsuite/integration/basic/pom.xml

```diff
@@ -163,6 +163,9 @@
                                         <include>org/jboss/as/test/integration/transaction/inflow/TransactionInflowTestCase.java</include>
                                         <include>org/jboss/as/test/integration/jsr77/**/*TestCase*.java</include>
                                     </includes>
+                                    <excludes>
```

Comment:
Why have you modified this file? This test is not listed in the includes list, so it will not run anyway.